lib/tandem/tandemSimulator.js

```diff
@@ -79,6 +84,13 @@ exports.make = function(config){
     basal: function(event){
       ensureTimestamp(event);
       if (currBasal != null) {
+        if(currBasal.deliveryType === 'temp') {
+          if (currTempBasal !== null) {
+            currBasal.percent = currTempBasal.percent;
```

Comment:
The percents aren't coming through anywhere. Check through all the steps to get them through? There's also an issue with the payloads - the issue that came up before with using the object builder's `set` method like a `with_*` method.